lib/supplejack/story.rb

```diff
@@ -97,6 +97,25 @@ def destroy
       end
     end
 
+    # Executes a POST request to the API with the Story ID, the user's api_key &
+    # an Array of Hashes for item positions ex: [{ id: 'storyitemid', position: 100 }]
+    #
+    # @return [ true, false ] True if the API response was successful, false if not.
+    #
+    def reposition_items(positions)
+      begin
```

Comment:
I think begin is redundant here since you are rescuing the whole method

Reply:
Thats true we need better linting in the client 😄

Reply:
Done!